dev-cpp/highway/files/0001-fix-compile-for-armv7-targets-with-vfp4-and-lower.patch

```diff
@@ -0,0 +1,115 @@
+From dc63f813c465f3bf95cb5b98f01aeed28b81173c Mon Sep 17 00:00:00 2001
```

Comment:
Please include a link to the patch source, and any other relevant references (e.g. Gentoo bugs), at the top of the patch file.

Reply:
The link to the gentoo bug is in the git commit msg, through which you can find the upstream issue linked. If thats insufficent I can add it right into the patch later.

Reply:
I usually like it in the patch itself as well because it makes it much faster to determine if a patch is needed still and, importantly, if other distros copy the patch, they get the benefit of the metadata.

Reply:
metadata added